massa-client/src/cmds.rs

```diff
@@ -415,253 +376,163 @@ impl Command {
                                 priva, publ, ad
                             ));
                         }
-                        repl_ok!(res)
                     }
                 }
+                if !json {
+                    println!("{}", res);
+                }
+                Ok(Box::new(()))
```

Comment:
@AureliaDolo `wallet_info` returns nothing currently, what kind of JSON output do we want?

Reply:
I made a follow-up issue https://github.com/massalabs/massa/issues/1825